packages/valory/contracts/dynamic_contribution/contract.py

```diff
@@ -110,20 +110,21 @@ def get_all_erc721_transfers(
         # Avoid parsing too many blocks at a time. This might take too long and
         # the connection could time out.
         MAX_BLOCKS = 300000
-        to_block = ledger_api.api.eth.get_block_number() if to_block == "latest" else to_block
+        to_block = (
+            ledger_api.api.eth.get_block_number() if to_block == "latest" else to_block
+        )
         ranges = list(range(from_block, to_block, MAX_BLOCKS)) + [to_block]
 
         entries = []
         for i in range(len(ranges) - 1):
             from_block = ranges[i]
-            to_block = ranges[i+1]
-            entries += factory_contract.events.Transfer.createFilter(
+            to_block = ranges[i + 1]
+            entries += factory_contract.events.Transfer.create_Filter(
```

Comment:
Are you sure it is not:

```suggestion
            entries += factory_contract.events.Transfer.create_filter(
```

?